internal/cmd/run/xcuitest.go

```diff
@@ -52,6 +53,7 @@ func NewXCUITestCmd() *cobra.Command {
 	sc.String("name", "suite.name", "", "Sets the name of the job as it will appear on Sauce Labs")
 	sc.String("app", "xcuitest.app", "", "Specifies the app under test")
 	sc.String("testApp", "xcuitest.testApp", "", "Specifies the test app")
+	sc.StringSlice("otherApps", "xcuitest.otherApps", []string{}, "Specifies the other apps")
```

Comment:
> "Specifies the other apps"
What "the" other apps? 😄 What happens with the other apps? The description field tries to convey some helpful message for the user to understand the purpose of this field. It's pretty clear that "other apps" is set when you set the flag, given its name. But it's unclear as to what that means. What does it do? What happens if I set it?

E.g.:
```suggestion
	sc.StringSlice("otherApps", "xcuitest.otherApps", []string{}, "Specifies any additional apps that are installed alongside the main app")
```